src/Parser/Core/Modules/Items/EonarsCompassion.js

```diff
@@ -0,0 +1,84 @@
+import SPELLS from 'common/SPELLS';
+import ITEMS from 'common/ITEMS';
+import React from 'react';
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import { formatNumber, formatPercentage } from 'common/format';
+
+const PANTHEON_MAX_SHIELD_PER_PROC = 4;
+
+class EonarsCompassion extends Analyzer {
+  static dependencies = {
+      combatants: Combatants,
+  };
+  trinketHealing = 0;
+  trinketProc = 0;
+  pantheonShieldHealing = 0;
+  pantheonProc = 0;
+  pantheonShieldCast = 0;
+
+  triggerBuffs = [
+    SPELLS.EONARS_COMPASSION_PANTHEONBUFF_RDROOD.id,
```

Comment:
Make it `EONARS_COMPASSION_PANTHEONBUFF_RDRUID`, leaving the 'colloquial' spelling is asking for an error somewhere down the line.